neuro_san/internals/run_context/langchain/llms/standard_langchain_llm_factory.py

```diff
@@ -406,4 +422,4 @@ def create_base_chat_model(self, config: Dict[str, Any]) -> BaseLanguageModel:
         else:
             raise ValueError(f"Class {chat_class} for model_name {model_name} is unrecognized.")
 
-        return llm
+        return LangChainLlmResources(llm, http_client=http_client)
```

Comment:
And finally, construct instance of LangChainLlmResources to return.
Note that http_client could be None here. In this case actual http connections release may not happen
when we are done with request.